qfieldsync/gui/checker_feedback_table.py

```diff
@@ -21,31 +21,24 @@
  ***************************************************************************/
 """
 
-from libqfieldsync.project_checker import ProjectCheckerFeedback
+from libqfieldsync.project_checker import Feedback, ProjectCheckerFeedback
+from qgis.core import QgsApplication
 from qgis.PyQt.QtCore import Qt
-from qgis.PyQt.QtWidgets import QLabel, QTableWidget, QTableWidgetItem
+from qgis.PyQt.QtWidgets import QLabel, QSizePolicy, QTableWidget, QTableWidgetItem
+
+from ..utils.qt_utils import make_icon
 
 
 class CheckerFeedbackTable(QTableWidget):
     def __init__(self, checker_feedback: ProjectCheckerFeedback, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self.setColumnCount(3)
-        self.setHorizontalHeaderLabels(
-            [self.tr("Level"), self.tr("Source"), self.tr("Message")]
-        )
+        self.setColumnCount(2)
+        self.setHorizontalHeaderLabels(["  ", self.tr("Message")])
```

Comment:
Is there a reason why the column header is two empty spaces? If yes, I guess it would be nice to add a small comment.

Reply:
Just meant as a way to insure a minimum width really. It can be empty too

Reply:
If we need the minimum width I am ok, just mention this in a comment, because it is not immediately clear.